### PR TITLE
Deploy docker image on CI

### PR DESCRIPTION
**Pull request by e00E**

### CODE REVIEW

File: docker/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -euo pipefail
+
+image_name=$1
```
Comment (fleupold) [inline docker/deploy.sh]: Isn't this the image tag not image name?

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# This is a dummy build to get the dependencies cached
+RUN cargo build --release
+# Copy sources
+COPY src src
+COPY contracts contracts
+RUN touch src/main.rs && cargo build --release
+
+FROM debian:stretch-slim
+RUN apt-get update && apt-get --yes install openssl
+COPY --from=builder /usr/src/app/target/release/etherbalance /bin/
```
Comment (fleupold) [inline docker/Dockerfile]: 👍Very nice way to keep the image size small.

File: docker/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+CMD etherbalance
```
Comment (fleupold) [inline docker/Dockerfile]: We should probably wrap this inside [tini](https://github.com/krallin/tini). We had an issue with our dex-services binary not handling signals well.

cf. https://github.com/gnosis/dex-services/blob/16ef8cf025c41b80318544df9a45890cb764deff/docker/rust/Dockerfile#L41-L44

File: docker/deploy.sh (hunk 1 above)
Comment (e00E) [reply docker/deploy.sh]: I copied the script from dex-services but I think you're right.

**Pull request merged**